cmake/modules/FindDiaSDK.cmake

```diff
@@ -69,6 +69,27 @@ if(NOT DiaSDK_ROOT)
 	endif()
 endif()
 
+# Gather up the include directories listed in the include environment variable.
+set(env_includes)
+if(CMAKE_HOST_SYSTEM_NAME STREQUAL "CYGWIN")
+	# INCLUDE is a list of windows style paths.
+	# This needs to be converted to cygwin style paths to feed to CMake.
+	if(CMAKE_CYGPATH AND (NOT "$ENV{INCLUDE}" STREQUAL ""))
```

Comment:
Shouldn't `env_includes` be set even if this condition is false?

Reply:
We set a default empty value on L73

Reply:
Right. I missed that.